src/LaunchDarkly.Client/LdClient.cs

```diff
@@ -132,7 +136,9 @@ public JToken JsonVariation(string key, User user, JToken defaultValue)
                 }
                 catch (Exception e)
                 {
-                    Logger.LogError("Exception caught when evaluating all flags: " + e.Message, e);
```

Comment:
Exception is passed in the end as a format argument.  If the actual exception message contained, {0}, this would get replaced inside it.